### PR TITLE
Defender PowerShell reference update

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
[1869916](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1869916)

90 new param added to 3 cmdlets.

### CODE REVIEW

**Comment by TimShererWithAquent** [comment]
Got input from Amir Dashti and the Defender team.

I'm going to keep this open over the weekend while we follow up on another Defender issue (that I don't think will affect this one, but I want to make sure), then I'll change if from draft.

**Review by ghost — APPROVED** [review]

File: docset/winserver2022-ps/defender/Add-MpPreference.md [hunk 1]
```diff
@@ -28,13 +31,20 @@ Use this cmdlet to add exclusions for file name extensions, paths, and processes
...
-```
-PS C:\> Add-MpPreference -ExclusionPath "C:\Temp"
+```powershell
+Add-MpPreference -ExclusionPath "C:\Temp"
 ```
 
 This command adds the folder C:\Temp to the exclusion list.
 The command disables Windows Defender scheduled and real-time scanning for files in this folder.
 
+### Example 2: All an application to access folders
```
Comment (get-itips) [inline docset/winserver2022-ps/defender/Add-MpPreference.md:41]:
```suggestion
### Example 2: Allow an application to access folders
```

File: docset/winserver2022-ps/defender/Remove-MpPreference.md [hunk 2]
```diff
@@ -45,6 +71,51 @@ This command will exclude only that specific file app.exe in that specific folde
 
 ## PARAMETERS
 
+### -AllowDatagramProcessingOnWinServer
+Removes whether to disable inspection of UDP connections on Windows Server.
```
Comment (get-itips) [inline docset/winserver2022-ps/defender/Remove-MpPreference.md:75]: Is _Removes_ the correct word here or maybe _Determines_? the same for the other parameters...

**Review by get-itips — CHANGES_REQUESTED** [review]
Please review the use of the word "removes" in multiple parameter descriptions, maybe you intended to use "Specifies" ?

**Review by dariomws — COMMENTED** [review]
It seems that there are some parameters missing yet:
- ThrottleForScheduledScanOnly, DisableNetworkProtectionPerfTelemetry, and TrustLabelProtectionStatus are missing
- SignaturesUpdatesChannel was updated to DefinitionUpdatesChannel in 2022

This applies to Set- and Remove-.

**Review by dariomws — APPROVED** [review]
@TimShererWithAquent sounds good to me. Thank you for your explanation!

**Review by get-itips — APPROVED** [review]

**Pull request merged**